project_kanban/models/project.py

```diff
@@ -19,3 +17,11 @@ def _compute_get_recent_date(self):
         string="Recent date",
         help="This will be auto set"
     )
+
+class ProjectTask(models.Model):
+    _inherit = 'project.task'
+
+    def write(self,vals):
+        res = super(ProjectTask, self).write(vals)
+        self.project_id.write_date = self.write_date
```

Comment:
No... is this the one approved by @Chandresh-SerpentCS ?

Reply:
Yes sir. We just need to update write_date of project by task write_date.